packages/runtime/test/transport/devices.test.ts

```diff
@@ -19,4 +19,14 @@ describe("Devices", () => {
         const sharedSecret = DeviceMapper.toDeviceSharedSecret(onboardingInfo);
         expect(sharedSecret).toBeDefined();
     });
+
+    test("should set the communication language", async () => {
+        const result = await transportServices1.devices.setCommunicationLanguage({ communicationLanguage: "fr" });
+        expect(result).toBeSuccessful();
+    });
+
+    test("should not set the communication language with an invalid language", async () => {
+        const result = await transportServices1.devices.setCommunicationLanguage({ communicationLanguage: "fra" });
+        expect(result).toBeAnError(/.*/, "error.runtime.devices.communicationLanguageNotISO639");
```

Comment:
You could add the error message here as a first parameter (I know we are not consistent with this in our existing tests, but it probably won't hurt to pay more attention to it.).